tests/suites/deploy/os.sh

```diff
@@ -0,0 +1,47 @@
+
+test_deploy_os() {
+    if [ "$(skip 'test_deploy_os')" ]; then
+        echo "==> TEST SKIPPED: deploy to os"
+        return
+    fi
+
+    (
+        set_verbosity
+
+        cd .. || exit
+
+
+
+        case "${BOOTSTRAP_PROVIDER:-}" in
+            "ec2")
+                run "run_deploy_centos"
+                ;;
+            *)
+                echo "==> TEST SKIPPED: deploy_centos - tests for LXD only"
```

Comment:
Shouldn't this echo 'tests for EC2 only"?

Reply:
yes, I have another commit to fix this and some other aws/ec2 issues across the board.  should be up in a minute.